specification/search/data-plane/Azure.Search/preview/2020-06-30-Preview/searchindex.json

```diff
@@ -275,6 +275,33 @@
               "name": "SearchOptions"
             }
           },
+          {
+            "name": "scoringStatistics",
+            "in": "query",
+            "type": "string",
+            "enum": [
+              "local",
+              "global"
+            ],
+            "x-ms-enum": {
```

Comment:
We've recently added explicit enum label names and descriptions to all `x-ms-enum` declarations. Please follow the same pattern here; There should be lots of examples in the specs now.

Also, if we think it's at all likely we might add new values besides `local` and `global` in the future, we should use `modelAsString: true` here.

Reply:
I dont think there's any plan on adding new values to this enum.